src/app/events/[slug]/components/EventSection.tsx

```diff
@@ -0,0 +1,55 @@
+import { MagnifyingGlass } from '@phosphor-icons/react/dist/ssr'
+
+import { Card } from '@/components/Card'
+import { CardGrid } from '@/components/CardGrid'
+import { PageSection } from '@/components/PageSection'
+
+import type { Event } from '../../types/eventType'
+import { getMetaData } from '../../utils/getMetaData'
+import { sortEventsAsc } from '../../utils/sortEvents'
+
+type EventSectionProps = NonNullable<Event['eventSection']>
+
+export function EventSection({ title, events }: EventSectionProps) {
+  const sortedEvents = sortEventsAsc(events)
```

Comment:
What do you think of passing already sorted events to `EventSection`? Meaning, moving the sorting to the parent?

Reply:
I'm ending very much up in TS hell - I'm going to move this forward and maybe we can have a look together on Monday.